data/successors/successor 1 introduction.txt

```diff
@@ -0,0 +1,1077 @@
+			`	"Hark... noble visitor! We bid thee... welcome and good tidings." The two eye you closely as they wait for a response.`
+			choice
+				`	"Hello. I am Captain <first> <last>."`
+				`	"I'm <last> of the <ship>. Pleased to meet you."`
+			`	The two once again proceed slowly and carefully, the speaker briefly conversing with the military officer before each phrase. "I am Sieasej... of House Sioeora among the Successors. Thou'rt a human, art thou not?"`
+			choice
+				`	"Yes, I am."`
+				`	"What else would I be?"`
+			`	This sparks a wave of color among the gathered members, most of all Sieasej, who spends several seconds in a vibrant shade of orange before returning to their resting hue. "Then i'tis confirmed. 'Tis a cause for much excitement, <last>, for thou'rt the first of humans that's been met by us in fifteen hundred years, since far before the time thy kind had even left the bounds of thine own planet's clouds. House Sioeora's honored t'be the one that gives thee greeting. So am I, Captain, for my part in it." The members of the group splay their free arms towards you and glisten with ripples of blue across their skin. You perform a short bow in return.`
+			label questions
+			choice
+				`	"What should I do now?"`
+					goto whatnow
+				`	"How do you know my language?"`
+					to display
+						not "Successors: language"
+					goto language
+				`	"How could you have met humans fifteen hundred years ago?"`
+					to display
+						not "Successors: history"
+					goto history
+				`	"What is House Sioeora?"`
+					to display
+						not "Successors: house"
+					goto house
+			label language
+			action
+				set "Successors: language"
+				set "Successors: history"
+			`	"T'was a time in which our Predecessors' home domains reach'd farther to the north, and farther still the reach of their intention to explore. For life, it is a rare and precious thing, and doubly so that life which is possess'd of sapience and culture, that's noble in reason, infinite in faculty, as thine. So then, our Predecessors waited under cloak of dark to silently observe thee in thy glimmering development. And if 'twere not from the rest of the stars so untimely ripp'd, we surely would have had the pleasure t'watch thy kind advance to what's its present point."`
+			`	Sieasej pauses for a moment, wriggling under their cloak. "I am a scholar of some olden languages," they say, "and of the worlds our Predecessors met so long ago. I'm one of very few who's learned thy tongue, and others, at least as 'twas gone fifteen hundred years ago."`
```

Comment:
This feels rather awkward. Maybe do this to start?
```suggestion
			`	Sieasej pauses for a moment, wriggling under their cloak. "I am a scholar of some olden languages," they say, "and of the worlds our Predecessors met so long ago. I happen to be one of very few who's learned thy tongue, and others, at least as 'twas gone fifteen hundred years ago."`
```

Reply:
You're right that the sentence was awkward; I've simplified it overall rather than add more words.

Reply:
Looks good.